draft-ietf-ppm-dap.md

```diff
@@ -1461,33 +1462,30 @@ ignore it. In addition, it MAY alert the Client with error `reportRejected`.
 The Leader MUST ignore any report pertaining to a batch that has already been
 collected (see {{input-share-validation}} for details). Otherwise, comparing
 the aggregate result to the previous aggregate result may result in a privacy
-violation. Note that this is also enforced by the Helper during the aggregation
-interaction. The Leader MAY also abort the upload interaction and alert the
-Client with error `reportRejected`.
+violation. The Leader MAY also abort with error `reportRejected`.
 
 The Leader MUST ignore any report whose timestamp is outside of the task's
-`time_interval`. When it does so, it SHOULD also abort the upload interaction
-and alert the Client with error `reportRejected`.
+`time_interval`. When it does so, it SHOULD also abort with error
+`reportRejected`.
```

Comment:
Brevity: the errors section makes it clear that "abort with error" means telling the client